packages/create-cloudflare/package.json

```diff
@@ -30,7 +30,6 @@
 		"dev:codemod": "node -r esbuild-register scripts/codemodDev.ts",
 		"check:lint": "eslint .",
 		"check:type": "tsc",
-		"lint": "eslint",
```

Comment:
we already have check:lint above this which actually lints the package, this command doesn't do anything